demo/seed-devnet.sh

```diff
@@ -72,14 +86,21 @@ function seedFaucet() {
 
 function publishReferenceScripts() {
   echo >&2 "Publishing reference scripts ('νInitial' & 'νCommit')..."
-  echo $(docker run --rm \
-    -v ${PWD}/devnet/credentials:/credentials:ro \
-    -v ${PWD}/devnet/ipc:/ipc \
-    ghcr.io/input-output-hk/hydra-node -- \
-    publish-scripts \
-    --network-id ${NETWORK_ID} \
-    --node-socket /ipc/node.socket \
-    --cardano-signing-key /credentials/faucet.sk)
+  if [[ -x ${HYDRA_NODE_PATH} ]]; then
+    echo $($HYDRA_NODE_PATH publish-scripts \
+      --network-id ${NETWORK_ID} \
+      --node-socket ${DEVNET_DIR}/ipc/node.socket \
+      --cardano-signing-key ${DEVNET_DIR}/credentials/faucet.sk)
+  else
+    echo $(docker run --rm \
+      -v ${PWD}/devnet/credentials:/credentials:ro \
+      -v ${PWD}/devnet/ipc:/ipc \
+      ghcr.io/input-output-hk/hydra-node -- \
+      publish-scripts \
+      --network-id ${NETWORK_ID} \
+      --node-socket /ipc/node.socket \
+      --cardano-signing-key /credentials/faucet.sk)
```

Comment:
Seems like you redid roughly what we just fixed & merged to `master` today :+1: You might want to have a look and rebase.. sorry for the conflict :)